examples/azuread/run.sh

```diff
@@ -0,0 +1,2 @@
+sudo jupyterhub -f ./sample_jupyter_config.py –-no-ssl –-log-level=DEBUG
```

Comment:
`--no-ssl` is deprecated since jupyterhub 0.7. I'd rather the docs leave out sudo, and let users do `sudo run.sh` if they need the elevated permissions.

Reply:
fixed